systems/controllers/iiwa_velocity_broadcaster.cc

```diff
@@ -0,0 +1,140 @@
+
+  AS << -0.23, -0.2, 0.05;
+  A0 << -0.23, -0.2, 0.05;
+
+  A1 << -0.23, -0.6, 0.05;
+  A2 << 0.23, -0.6, 0.05;
+
+  A3 << 0.23, -0.2, 0.05;
+  
+  A4 << 0.23, -0.2, 0.05;
+
+  A5 << 0.23, -0.6, 0.05;
+
+  A6 << -0.23, -0.6, 0.05;
+
+  A7 << -0.23, -0.2, 0.05;
+
+  A8 << -0.23, -0.2, 0.05;
+  A9 << -0.23, -0.2, 0.05;
+
+  points[0] = AS;
+  points[1] = A0;
+  points[2] = A1;
+  points[3] = A2;
+  points[4] = A3;
+  points[5] = A4;
+  points[6] = A5;
+  points[7] = A6;
+  points[8] = A7;
+  points[9] = A8;
+  points[10] = A9;
+
+
+  auto ee_trajectory = drake::trajectories::PiecewisePolynomial<double>::FirstOrderHold(times, points);
+
+  // Adding status subscriber and receiver blocks
+  auto status_subscriber = builder.AddSystem(
+    drake::systems::lcm::LcmSubscriberSystem::Make<drake::lcmt_iiwa_status>(
+      "IIWA_STATUS", &lcm));
+  auto status_receiver = builder.AddSystem<drake::examples::kuka_iiwa_arm::IiwaStatusReceiver>();
+
+  // Adding Velocity Controller block
+  auto velocity_controller = builder.AddSystem<systems::KukaIiwaVelocityController>();
+
+  // Adding position controller block
+  auto position_controller = builder.AddSystem<systems::KukaIiwaPositionController>();
+
+  // Adding Trajectory Source
+  // TODO: create ee_trajectory
+  auto input_trajectory = builder.AddSystem<drake::systems::TrajectorySource>(ee_trajectory);
+
+  // Adding command publisher and broadcaster blocks
+  auto command_sender = builder.AddSystem<drake::examples::kuka_iiwa_arm::IiwaCommandSender>();
+  auto command_publisher = builder.AddSystem(
+    drake::systems::lcm::LcmPublisherSystem::Make<drake::lcmt_iiwa_command>(
+      "IIWA_COMMAND", &lcm));
+  // Setting command publisher publish period
+  command_publisher->set_publish_period(1.0/200.0);
+
+
+  builder.Connect(status_subscriber->get_output_port(),
+                  status_receiver->get_input_port(0));
+  builder.Connect(status_receiver->get_position_commanded_output_port(),
+                  velocity_controller->get_input_port(0));
+  builder.Connect(status_receiver->get_velocity_estimated_output_port(),
+                  velocity_controller->get_input_port(1));
+  
+  //Connecting q input from status receiver to position controller
+  builder.Connect(status_receiver->get_position_measured_output_port(),
+                  position_controller->get_input_port(0));
+  //Connecting x_desired input from trajectory to position controller
+  builder.Connect(input_trajectory->get_output_port(),
+                  position_controller->get_input_port(1));
+
+  //Connecting position controller to trajectory/velocity controller;
+  builder.Connect(position_controller->get_output_port(0),
+                  velocity_controller->get_input_port(2));
+
+  //Connecting desired orientation from position controller to trajectory/velocity controller;
+  builder.Connect(position_controller->get_output_port(1),
+                  velocity_controller->get_input_port(3));
```

Comment:
Hard-coded port numbering will eventually cause confusion. See here for an example of how to create getters for specific ports: https://github.com/DAIRLab/dairlib/blob/704bbf9db6aebf148a9806a537ef4f95d00f471e/systems/sensors/sim_cassie_sensor_aggregator.h#L35